test/configs/windows.utsc

```diff
@@ -3,6 +3,10 @@
     "test\\*.uts",
     "scapy\\contrib\\*.uts"
   ],
+  "remove_testfiles": [
```

Comment:
Can you add the same lines to osx.utsc too ?

Reply:
done